### DISCUSSION TITLE
How to train non-periodic structures

### DISCUSSION THREAD

**Discussion by caolz** [deepmodeling/deepmd-kit#427] [Q&A]
I want to know how to train non-periodic structures such as clusters, can I not provide the box.raw file?

**Comment by njzjz** [comment]
You can put an empty `nopbc` file under the system directory.

box file may be necessary but will be never used.

See the example here https://github.com/tongzhugroup/NNREAX/blob/master/dodecane/dodecane_v1.tar.bz2

**Reply by caolz** [reply]
Thanks for the reply, I get it.
